python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -521,6 +523,50 @@ def is_graph_backed_asset(self) -> bool:
     def is_executable(self) -> bool:
         return self._asset_node_snap.is_executable
 
+    def get_materialization_status_for_asset_health(self, graphene_info: ResolveInfo):
```

Comment:
What do you think about moving these helper functions to the `GrapheneAssetHealth` object? (Since they'll only be used by the asset health resolver presumably)

Reply:
that's where i had them originally, but when implementing i found i needed to call some of the functions in the `GrapheneAsset` class so i moved them here